Source/Core/InputCommon/InputConfig.h

```diff
@@ -31,6 +31,7 @@ class InputConfig
 	ControllerEmu* GetController(int index);
 	void ClearControllers();
 	bool ControllersNeedToBeCreated() const;
+	bool IsControllerControlledBySomeTypeOfGamepadDevice(int index) const;
```

Comment:
`SomeTypeOfGamepadDevice` includes joysticks, steering wheels, vive controllers, wiimotes, etc.

I put it there for a reason.